src/components/reusable/table/story-helpers/column-setting.ts

```diff
@@ -0,0 +1,234 @@
+import { LitElement, html, css, PropertyValueMap } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import { repeat } from 'lit/directives/repeat.js';
+
+import '@kyndryl-design-system/shidoka-foundation/components/button';
+import '@kyndryl-design-system/shidoka-foundation/components/icon';
+import lockedIcon from '@carbon/icons/es/locked/16';
+import unlockedIcon from '@carbon/icons/es/unlocked/16';
+
+import '../../globalFilter';
+import '../index';
+import '../../checkbox';
+
+@customElement('story-column-setting')
+class StoryColumSetting extends LitElement {
+  static override styles = css`
+    .min-max-width-100 {
+      --kyn-td-min-width: 100px;
+      --kyn-td-max-width: 100px;
+      --kyn-td-width: 100px;
+    }
+    .min-max-width-th-100 {
+      --kyn-th-min-width: 100px;
+      --kyn-th-max-width: 100px;
+      --kyn-th-width: 100px;
+    }
```

Comment:
IMO here `--kyn-th-width: 100px;` is enough unless it's shrinks or override by parent. Curious why we used min, max and width same px;

Reply:
This is a component built specially as a part of a story, not a real component. And I remember having all three to showcase a dev, asking a way to set column width; and this is one of the ways.

Reply:
got it